bubblewrap.c

```diff
@@ -1458,6 +1485,9 @@ main (int argc,
   /* We want sigchild in the child */
   unblock_sigchild ();
 
+  if (label_exec(opt_exec_label) == -1)
```

Comment:
This will mark the main process. However, what if pid namespaces are in use, and there is a pid1 (do_init()) in the sandbox. This will not be marked with the selinux context, but is still visible in the sandbox. I don't know if this is a problem or not though...